packages/patternfly-3/patternfly-react-extensions/less/catalog-tile.less

```diff
@@ -67,3 +67,55 @@
     overflow-y: hidden;
   }
 }
+
+.catalog-tile-view-pf-category {
+  margin: 10px 0;
+}
+
+.catalog-tile-view-pf-category-header {
+  padding-right: 20px;
+}
+
+.catalog-tile-view-pf-category-header-title {
+  font-size: 14px;
+}
+
+.catalog-tile-view-pf-category-view-all {
+  font-size: 12px;
+  margin-top: 2px;
+  overflow-x: hidden;
```

Comment:
why are `overflow-x` and `text-overflow` needed here? doesn't look like there's a `width` or anything that would cause the ellipsis to show.